app/controllers/page_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PageController < ApplicationController
+  def test
```

Comment:
can use `index` instead of `test`. And put a space on top and bottom of this action